webui/react/src/utils/color.ts

```diff
@@ -1,35 +1,61 @@
 import { GLASBEY } from 'constants/colors';
 
+/*
+ * h - hue between 0 and 360
+ * s - saturation between 0.0 and 1.0
+ * l - lightness between 0.0 and 1.0
+ */
 interface HslColor {
   h: number;
   l: number;
   s: number;
 }
 
+/*
+ * r - red between 0 and 255
+ * g - green between 0 and 255
+ * b - blue between 0 and 255
+ * a - alpha between 0.0 and 1.0
+ */
+interface RgbaColor {
```

Comment:
are there libraries we could use that do these color conversions for us?

Reply:
this one looks promising but it adds a decent amount in size (7.6kB) for something so small...

https://www.npmjs.com/package/color
https://bundlephobia.com/result?p=color@3.1.3

Reply:
hmm since you've added a test for it as well using our own implementation sgtm